mongodb/map.jinja

```diff
@@ -1,14 +1,19 @@
 {% set mongodb = salt['grains.filter_by']({
     'default': {
-        'pkg': 'mongodb',
-        'service': 'mongodb',
-        'conf_file': '/etc/mongodb/mongodb.conf',
-        'config': {
-            'key': 'value',
-        },
+      'key': '7F0CEB10',
+      'keyserver': 'keyserver.ubuntu.com',
+      'pkgs': [
+        {'mongodb-10gen': '*'},
```

Comment:
The `pkgs` value should probably also be `os_family` specific, unless they have the same default name for the package (e.g. `mongodb-server`) across distros.